vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -110,3 +118,24 @@
     retries: 10
     delay: 20
     until: provisioned_bmh.stdout ==  NUMBER_OF_BMH
+
+  ############### Pivot back to source cluster ##############
+
+  # Remove Ironic from target cluster
+  - name: Remove Ironic from target cluster
+    k8s:
+      name: metal3-ironic
+      kind: Deployment
+      state: absent
+      namespace: "{{ NAMESPACE }}"
+    environment:
+      KUBECONFIG: "/tmp/target.yaml"
+
+  - name: Install Ironic in Source cluster
+    shell: "bash /home/$USER/go/src/github.com/metal3-io/baremetal-operator/tools/run_local_ironic.sh"
+    environment:
+      CONTAINER_RUNTIME: "{{ CONTAINER_RUNTIME }}"
```

Comment:
this is specific to Kind, if someone would attempt to run this with minikube it would fail

Reply:
Actually we have explicitly mentioned it here https://github.com/metal3-io/metal3-dev-env/tree/master/scripts/feature_tests to use kind

Reply:
well it is not mentioned that it should not be run with Minikube, it just says that it defaults to Kind

Reply:
and until now there wasn't really anything preventing from running it with Minikube, was there ?

Reply:
TBH, feature test is not tested at all with minikube, so I cannot specifically say that it would work or not, since the entity which is calling this move.yml is not supposed to call it with minikube.

Reply:
Then please make it clear that it shall not be run with Minikube. Saying that it defaults to Kind is not clear enough.

Reply:
I have added additional code and check for minikube, pleas review